src/seedu/addressbook/AddressBook.java

```diff
@@ -22,6 +22,7 @@
 import java.util.Optional;
 import java.util.Scanner;
 import java.util.Set;
+import java.util.*;
```

Comment:
Wildcard imports should be avoided. Read up on **rule 3** of the Java [coding standards](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#package-and-import-statements).